_posts/2011-11-08-odata-service-validation-tool-update-19-new-rules.html

```diff
@@ -20,5 +20,5 @@
 </ul>
 <p>This rule update brings the total number of rules in the validation tool to <strong>97.</strong></p>
 <p><a href="http://odatavalidator.codeplex.com/">OData Service Validation Codeplex project</a>  is also updated with all recent changes along with documentation on <a href="http://odatavalidator.codeplex.com/wikipage?title=Authoring%20Rules&amp;referringTitle=Documentation">how to extend the validation service by authoring rules</a>.</p>
-<p>Keep validating and let us know what you think either on the <a href="https://www.odata.org/join-the-odata-discussion/">mailing list</a> or on the <a href="http://odatavalidator.codeplex.com/discussions">discussions page</a> on the Codeplex site.</p>
+<p>Keep validating and let us know what you think either on the <a href="/join-the-odata-discussion/">mailing list</a> or on the <a href="http://odatavalidator.codeplex.com/discussions">discussions page</a> on the Codeplex site.</p>
```

Comment:
The http://odatavalidator.codeplex.com/discussions link seems to be dead.

Reply:
@habbes Fixing dead links that point to external resources was beyond the scope of this pull request. In any event, I didn't have a valid alternative I could replace them with.